_data/accessibility-tests/select.yml

```diff
@@ -0,0 +1,112 @@
+# Keyboard navigation tests
+  - summary: All features of the select element are keyboard operable.
+    summary_additional: |
+      When using a keyboard,
+      you are able to tab into the select element, arrow down to select an option, and hit enter to select.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.1.1
+  - summary: The focus indicator appears around every element in the select component.
+    summary_additional: |
+      When navigating to select,
+      you can see the focus indicator appear around the select component and on options within the select dropdown.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.4.7
+  - summary: Navigating to select doesn't trigger a change of context.
+    summary_additional: |
+      When you type into the select,
+      the context of the select component doesn't change.
+      For example, landing on select doesn't cause a window to open or focus to jump to another field.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 3.2.1
+# Screen reader tests
+  - summary: Screen reader announces select dropdown items in a list.
+    summary_additional: |
+      When you navigate with arrow keys,
+      `enter` and `spacebar` key while using a screen reader, you hear the the drop down label and items in the list.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.1
+  - summary: Select options are read in the same order that they appear on the page.
+    summary_additional: |
+      When you use a screen reader and navigate through the dropdown list,
+      you hear each text option in the same order in which they appear on the page.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.2
+  - summary: Screen reader provides information about the purpose of the select.
+    summary_additional: |
+      When you are using a screen reader,
+      you will know the common purpose of the select option (e.g., date, time, state).
+    test_status: conditional
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.5
+  - summary: Screen reader announces user is in a combo box with select option.
+    summary_additional: |
+      When you use a screen reader and access the select component
+      you will hear that you have entered a combo box and that you can select an option.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 2.4.6
+  - summary: Content doesn't change until the user takes an action to change it.
+    summary_additional: |
+      When using a screen reader to navigate to the select,
+      if you don't take a prompted action, the content stays the same.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 3.2.2
```

Comment:
Is "Headings and labels" the right criterion for "Content doesn't change until the user takes an action to change it."? It seems off to me, so I'm tagging @amycole501 and @alex-hull. Also, even though the button reads "3.2.2" the link actually directs you to WCAG 2.4.6. Hoping the a11y team can identify the right fix. Thanks! cc @amyleadem

Reply:
Good catch! Yes, the criterion number is correct, but should link you to 3.2.2: [on input](https://www.w3.org/WAI/WCAG22/Understanding/on-input.html). Are y'all able to update, @amyleadem @mahoneycm

Reply:
It looks like `3.2.2` just needs to be added to the `wcag_details`. It was added in other PRs but hasn't made it's way onto this one yet

Reply:
Good catch! I updated this in [404181b](https://github.com/uswds/uswds-site/pull/2702/commits/404181bd54a9a7ae29c23090e169c2bf182f25b3)

Reply:
@amyleadem do you need anything from me here?

Reply:
Just need to confirm that this item now points to the correct link for 3.2.2 (Before it pointed to 2.4.6)

Reply:
It points correctly.